example/otlp/go.mod

```diff
@@ -0,0 +1,13 @@
+module github.com/wtong98/otel-test
+
+go 1.14
+
+require (
+	github.com/golang/protobuf v1.3.5 // indirect
```

Comment:
Please run `go mod tidy` to clean up go.mod.

Reply:
Done